### DISCUSSION TITLE
Webview2 custom browser with Manifest v3 upgrade

### DISCUSSION THREAD

**Discussion by AnoopLal91** [MicrosoftEdge/WebView2Feedback#2691] [Q&A]
Hi All,

Do we expect any impact on application using Webview2 custom browser with the Manifest v3 upgrade that coming on Jan 2023. 
In the current browser app, we are not calling any extension explicitly, and by default, browser extensions are not configurable in WebView2. Even then , when custom browser app is built, could see the reference of manifest V2(manifest.json file in Unindexed Rules folder).

_{
  "manifest_version": 2,
  "name": "Subresource Filter Rules",
  "ruleset_format": 1,
  "version": "10.34.0.24"
}

Note : Custom browser app is created for interaction with a RPA tool, UiPath without the use of any extension.

**Comment by victorthoang** [comment]
hello @AnoopLal91 

Are you still looking for an answer to your question? If you still need a response, we can turn this into an Issue with a Question tag (which is typically how we address questions and bugs.

The discussion thread might be more useful for off topic questions or non-issue related inquiries!

Let me know if you'd like me to create an issue from this discussion, or if you'd prefer to add it. Thanks!

P.S. I vaguely remember a discussion similar to this and I believe Nic had a response to this in the github issues thread.

**Comment by AnoopLal91** [comment]
yes @victorthoang , please help me to turn this into an issue.
@tonyantony33 FYI,
